### PR TITLE
Fix Kubernetes LeaseActor and improve logging

### PR DESCRIPTION
**Pull request by Arkatufus**
Related to https://github.com/akkadotnet/akka.net/issues/7379

## Changes

* Move `LeaseActor` from /user to /system
* Improve log messages

### CODE REVIEW

File: src/coordination/kubernetes/Akka.Coordination.KubernetesApi/KubernetesLease.cs [hunk 1]
```diff
@@ -66,7 +66,7 @@ public KubernetesLease(ExtendedActorSystem system, AtomicBoolean leaseTaken, Lea
             var client = new KubernetesApiImpl(system, kubernetesSettings);
             _timeout = _settings.TimeoutSettings.OperationTimeout;
             _leaseName = MakeDns1039Compatible(settings.LeaseName);
-            _leaseActor = system.ActorOf(
+            _leaseActor = system.SystemActorOf(
```
Comment (Arkatufus) [inline src/coordination/kubernetes/Akka.Coordination.KubernetesApi/KubernetesLease.cs:69]: Actual fix, make sure the `LeaseActor` lives under `/system`, not `/user`

**Review by Arkatufus — COMMENTED** [review]
Self-review

File: src/coordination/kubernetes/Akka.Coordination.KubernetesApi/KubernetesLease.cs (hunk 1 above)
Comment (Aaronontheweb) [reply src/coordination/kubernetes/Akka.Coordination.KubernetesApi/KubernetesLease.cs:69]: LGTM

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**